frontend/src/pages/projects/screens/spawner/storage/ClusterStorageEditModal.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
```

Comment:
this file is just the placeholder until dipanshu merges right?

Reply:
Yes, I think he can use it and expand upon it though.